### DISCUSSION TITLE
Error in docker-compose up

### DISCUSSION THREAD

**Discussion by zhongli1990** [luolingchun/flask-openapi3#59] [Q&A]
Hi, when I am trying to brought it up in a docker, it generated this error:

pydantic.errors.ConfigError: field "items_ForwardRef('Schema')" not yet prepared so type is still a ForwardRef, you might need to call Schema.update_forward_refs().

The source line is 
@app.get("/get2", tags=[aibot_tag])

Not sure anything wrong in my code?

========================================
chatbot-workflow   | Traceback (most recent call last):
chatbot-workflow   |   File "app.py", line 102, in <module>
chatbot-workflow   |     @app.get("/get2", tags=[aibot_tag])
chatbot-workflow   |   File "/usr/local/lib/python3.7/site-packages/flask_openapi3/scaffold.py", line 170, in decorator
chatbot-workflow   |     method=HTTPMethod.GET
chatbot-workflow   |   File "/usr/local/lib/python3.7/site-packages/flask_openapi3/openapi.py", line 286, in _do_decorator
chatbot-workflow   |     get_responses(combine_responses, extra_responses, self.components_schemas, operation)
chatbot-workflow   |   File "/usr/local/lib/python3.7/site-packages/flask_openapi3/utils.py", line 264, in get_responses
chatbot-workflow   |     "items": {"$ref": f"{OPENAPI3_REF_PREFIX}/{UnprocessableEntity.__name__}"}
chatbot-workflow   |   File "pydantic/main.py", line 404, in pydantic.main.BaseModel.__init__
chatbot-workflow   |   File "pydantic/main.py", line 1040, in pydantic.main.validate_model
chatbot-workflow   |   File "pydantic/fields.py", line 723, in pydantic.fields.ModelField.validate
chatbot-workflow   |   File "pydantic/fields.py", line 899, in pydantic.fields.ModelField._validate_singleton
chatbot-workflow   |   File "pydantic/fields.py", line 699, in pydantic.fields.ModelField.validate
chatbot-workflow   | pydantic.errors.ConfigError: field "items_ForwardRef('Schema')" not yet prepared so type is still a ForwardRef, you might need to call Schema.update_forward_refs().

**Comment by luolingchun** [comment]
Can you provide more infomation？
- flask-openapi3 version
- more source code in your demo
- pydantic version
- flask version

**Comment by zhongli1990** [comment]
Hi, I think my approach might not be working with this package, so I restarted the path from scratch and got a new question and will raise it in a different thread. Happy to close it for now.
